Dockerfile

```diff
@@ -11,15 +11,16 @@ ENV PYTHONDONTWRITEBYTECODE=1
 ADD requirements.txt /builder/requirements.txt
 
 WORKDIR /builder
-RUN apt-get update && apt-get install gcc -y
+RUN apt-get update && apt-get install gcc libpq-dev -y
+RUN pip install --upgrade pip && pip install --user -r requirements.txt
```

Comment:
We have to do it, once on line 16. We remove the compiler (`gcc` and libraries) used by some packages installed by pip (some packages require `gcc`, such as `psycopg2`.

We install the required compilers and os (gcc, libpq-dev), install python libraries than clean up from the docker image the gcc and sub-dependencies. libpq-dev needs to stay because psycopg2 uses some linked libraries.